clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -1282,9 +1303,9 @@ trueDualPortBlockRam#, trueDualPortBlockRamWrapper ::
 
   -> (Signal domA a, Signal domB a)
   -- ^ Outputs data on /next/ cycle. If write enable is @True@, the data written
-  -- will be echoed. If write enable is @False@, the read data is returned. If
-  -- port enable is @False@, it is /undefined/.
-trueDualPortBlockRam# wmA wmB clkA enA weA addrA datA clkB enB weB addrB datB
+  -- will be echoed. If write enable is @False@, the read data is returned.
+trueDualPortBlockRam# wmA wmB clkA enA weA addrA datA
+ clkB enB weB addrB datB
```

Comment:
I'd prefer the continuation of the line to be indented by four spaces rather than one.